docs/transparent-validation/00_templates/laboratory-stand-assembly-guide-template.md

```diff
@@ -0,0 +1,124 @@
+# Laboratory stand dedicated to MSI PRO Z690-A assembly guide
+
+## Introduction
+
+This document describes the assembly procedure dedicated to [DEVICE_NAME]
+testing stand.
+
+## Prerequisites
+
+The below table contains information about all elements which are needed to
+create the testing stand.
+
+* [device]
+* [element1]
+* [element2]
+
+## Pre-setup activities
+
+The following subsections describe the method of preparing all the
+components of the laboratory stand.
+
+### [DEVICE_NAME]
+
+[DEVICE_NAME] platform should be prepared in accordance with the
+[Assembly](some.md#some-section) documentation.
+
+### [EG.] RTE
+
+RTE (acronym: Remote Testing Environment) should be prepared in accordance with
+[Quick start guide](../rte/v1.1.0/quick-start-guide.md) documentation dedicated
+to the device.
+
+### [EG.] Sonoff
+
+Sonoff should be prepared in accordance with
+[Quick start guide](../sonoff/quick-start-guide.md) documentation dedicated to
+the device.
+
+## Connections
+
+The following sections describe how to enable all of the following features:
+
+* serial connection to the platform,
+* controlling power supply,
+* enabling basic power actions with the platform (power off/power on/reset),
+* external flashing with the RTE.
+* [optional-connections]
+
+### Serial connection
+
+1. ...
```

Comment:
This subsection should contain any information about what it concerns.

Reply:
Updated